packages/e2e-tests/specs/experiments/blocks/comments-query.test.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	activateTheme,
+	clickBlockAppender,
+	createNewPost,
+	getEditedPostContent,
+	insertBlock,
+	setOption,
+} from '@wordpress/e2e-test-utils';
+
+describe( 'Comment Query Loop', () => {
+	beforeAll( async () => {
+		await activateTheme( 'emptytheme' );
+		await setOption( 'page_comments', true, 'options-discussion.php' );
+		await setOption( 'comments_per_page', '2', 'options-discussion.php' );
```

Comment:
It seems like we're not really testing for any behavior based on these settings now?

Reply:
True, at least while the unit test is the one that checks the pagination behavior. Let's make an e2e to check the navigation links and these settings and functions will be necessary.